components/__init__.py

```diff
@@ -342,13 +342,14 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['./configure --prefix="$SPDK_PREFIX" ' \
+                commands=['./configure --prefix="$SPDK_PREFIX" --with-shared ' \
                           ' --with-fio="$FIO_SRC"',
                           'make $JOBS_OPT', 'make install',
                           'cp dpdk/build/lib/* "$SPDK_PREFIX/lib"',
                           'mkdir -p "$SPDK_PREFIX/share/spdk"',
                           'cp -r include scripts examples/nvme/fio_plugin ' \
                           '"$SPDK_PREFIX/share/spdk"'],
+		patch='$PATCH_PREFIX/spdk_export_set_thread.patch',
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (add 14 spaces).

Reply:
Should fix this one.  You can run ./check_modules.sh -c to check internal ones to see if you fix it locally

Reply:
yes, sure.